Bisulfite Sequencing Analysis Tools
=

Quality Control
-
* BSeQC		--- http://bioinformatics.oxfordjournals.org/content/29/24/3227 --- SAM/BAM files, 
* trim_galore	--- for RRBS experiments
* FastQC		--- additional quality control
* MethyQA		--- FastQC + BRAT[tool] alignment, methylation ratios at cytosine sites --- http://www.biomedcentral.com/1471-2105/14/259


Mapper
-

* Bismark		--- seems to be the tool to use --- http://bioinformatics.oxfordjournals.org/content/27/11/1571 --- http://www.bioinformatics.babraham.ac.uk/projects/bismark/
* Bison		--- similar to bismark, runs on a cluster, needs MPI --- http://sourceforge.net/projects/dna-bison/
* BS Seeker	--- http://www.biomedcentral.com/1471-2105/11/203 --- http://pellegrini.mcdb.ucla.edu/BS_Seeker/BS_Seeker.html
* BSmap		--- https://code.google.com/p/bsmap/ (Jan 2013)
* segemehl 	--- with gaps --- http://www.bioinf.uni-leipzig.de/Software/segemehl/ --- http://bioinformatics.oxfordjournals.org/content/28/13/1698.full
* Bicycle		--- from mapping to methylcytosine --- http://sing.ei.uvigo.es/bicycle/ 



After Mapping
-

* GBSA		--- input from BSmap, BS Seeker, RRBSmap --- http://nar.oxfordjournals.org/content/41/4/e55.long --- http://ctrad-csi.nus.edu.sg/gbsa/
* methylKit 	--- https://code.google.com/p/methylkit/



Pipeline
-
* *MethPipe*	--- pretty new --- http://smithlab.usc.edu/methpipe/
* BiQ Analyzer HT --- Java Web Start --- http://biq-analyzer-ht.bioinf.mpi-inf.mpg.de/  ---  http://nar.oxfordjournals.org/content/39/suppl_2/W551.full



Other
-

* MethylCoder	---
* BISMA		--- 
* BSmooth		--- Don't think it's up-to-date - http://rafalab.jhsph.edu/bsmooth/




*** MISC ***
- RRBS - Reduced representation bisulfite sequencing
- http://www.bioinformatics.babraham.ac.uk/projects/bismark/RRBS_Guide.pdf
