Bisulfite Sequencing Analysis Tools
=

Quality Control
-

* **BSeQC**		--- SAM/BAM files | [Publication](http://bioinformatics.oxfordjournals.org/content/29/24/3227)  
* **trim_galore** --- for RRBS experiments
* **FastQC**		--- additional quality control
* **MethyQA**		--- FastQC + BRAT[tool] alignment, methylation ratios at cytosine sites | [Publication](http://www.biomedcentral.com/1471-2105/14/259)


Mapper
-

* **Bismark**		--- seems to be the tool to use | [Publication](http://bioinformatics.oxfordjournals.org/content/27/11/1571) | [URL](http://www.bioinformatics.babraham.ac.uk/projects/bismark/)
* **Bison**		--- similar to bismark, runs on a cluster, needs MPI | [URL](http://sourceforge.net/projects/dna-bison/)
* **BS Seeker**	--- [Publication](http://www.biomedcentral.com/1471-2105/11/203) | [URL](http://pellegrini.mcdb.ucla.edu/BS_Seeker/BS_Seeker.html)
* **BSmap**		--- [URL](https://code.google.com/p/bsmap/) (Jan 2013)
* **segemehl** 	--- with gaps | [Publication](http://bioinformatics.oxfordjournals.org/content/28/13/1698.full) | [URL](http://www.bioinf.uni-leipzig.de/Software/segemehl/)
* **Bicycle**		--- from mapping to methylcytosine | [URL](http://sing.ei.uvigo.es/bicycle/)



After Mapping
-

* **GBSA**		--- input from BSmap, BS Seeker, RRBSmap | [Publication](http://nar.oxfordjournals.org/content/41/4/e55.long) | [URL](http://ctrad-csi.nus.edu.sg/gbsa/)
* **methylKit** 	--- [URL](https://code.google.com/p/methylkit/)



Pipeline
-

* **MethPipe**	--- pretty new | [URL](http://smithlab.usc.edu/methpipe/)
* **BiQ Analyzer HT** --- Java Web Start | [Publication](http://nar.oxfordjournals.org/content/39/suppl_2/W551.full) | [URL](http://biq-analyzer-ht.bioinf.mpi-inf.mpg.de/)



Other
-

* **MethylCoder**	---
* **BISMA**		--- 
* **BSmooth**		--- Don't think it's up-to-date | [URL](http://rafalab.jhsph.edu/bsmooth/)

MISC
-

- RRBS - Reduced representation bisulfite sequencing
- http://www.bioinformatics.babraham.ac.uk/projects/bismark/RRBS_Guide.pdf
